scio-avro/src/main/scala/com/spotify/scio/avro/types/AvroType.scala

```diff
@@ -231,7 +231,7 @@ object AvroType {
  *
  * This decouples generated fields and methods from macro expansion to keep core macro free.
  */
-class AvroType[T: TypeTag] {
+class AvroType[T: TypeTag] extends Serializable {
```

Comment:
Where is `AvroType` pulled into a lambda?

Reply:
In the function that converts typed objects to generic record. https://github.com/spotify/scio/pull/1089/files#diff-417755bd55a6dbb7a50a9f58d6a7af4bL973